app/validations/userValidations.js

```diff
@@ -0,0 +1,26 @@
+const { check } = require('express-validator');
+
+exports.validateLogin = [
+  check('email', 'Email requerido').notEmpty(),
+  check('email', 'Email no válido').isEmail(),
+  check('email', 'Email no válido').custom(email => {
+    if (
+      !email
+        .trim()
+        .toLowerCase()
+        .includes('@wolox')
+    )
+      throw new Error('Email no válido, no pertenece a Wolox');
```

Comment:
I think this error is catched by express-validator, am i right? I ask because of the custom error thing.

Reply:
Yes!